manifests/integrations/kubernetes.pp

```diff
@@ -0,0 +1,45 @@
+# Class: datadog_agent::integrations::kubernetes
+#
+# This class will install the necessary configuration for the kubernetes integration
+#
+# Parameters:
+#   $url:
+#     The URL for kubernetes API
+#
+#   $tags:
+#     optional array of tags
+#
+#
+#
+# Sample Usage:
+#
+#   class { 'datadog_agent::integrations::kubernetes' :
+#     api_server_url  => 'https://kubernetes:443',
+#   }
+#
+class datadog_agent::integrations::kubernetes(
+  $api_server_url = 'Enter_Your_API_url',
+  $tags = []
+) inherits datadog_agent::params {
+  include datadog_agent
+
+  if $::datadog_agent::agent6_enable {
+    $dst = "${datadog_agent::conf6_dir}/kubernetes.yaml"
+  } else {
+    $dst = "${datadog_agent::conf_dir}/kubernetes.yaml"
+  }
+
+  package { 'dd-check-kubernetes':
```

Comment:
This is not always true, we bundle a check with the agent, so what you should ensure here is the agent is installed not `dd-check-kubernetes` (I think that's actually taken care of by the `include datadog-agent` directive above).